sql/core/src/test/resources/sql-tests/inputs/cte-nonlegacy.sql

```diff
@@ -0,0 +1,115 @@
+create temporary view t as select * from values 0, 1, 2 as t(id);
+create temporary view t2 as select * from values 0, 1 as t(id);
+
+-- CTE non-legacy substitution
+SET spark.sql.legacy.ctePrecedence.enabled=false;
```

Comment:
use `--SET spark.sql.legacy.ctePrecedence.enabled=false`, so that this won't be treated as a query and appear in the result query.